app/models/pipeline_run.rb

```diff
@@ -393,6 +395,15 @@ def succeeded?
     job_status == STATUS_CHECKED
   end
 
+  def db_load_input_validations
+    file = Tempfile.new
+    downloaded = PipelineRun.download_file_with_retries(s3_file_for("input_validations"),
+                                                        file.path, 3, false)
+    error_message = downloaded ? JSON.parse(File.read(file))["Validation error"] : nil
+    update(error_message: error_message) if error_message
```

Comment:
may want to append to existing error_message if we use this field for other messages in the future

Reply:
good point